backend/app/crud/config/config.py

```
@@ -0,0 +1,159 @@
+            return config, version
+
+        except Exception as e:
+            self.session.rollback()
+            logger.error(
+                f"[ConfigCrud.create] Failed to create configuration | "
+                f"{{'name': '{config_create.name}', 'project_id': {self.project_id}, 'error': '{str(e)}'}}",
+                exc_info=True,
+            )
+            raise HTTPException(
+                status_code=500,
+                detail=f"Unexpected error occurred: failed to create config",
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)